.gitignore

```diff
@@ -6,4 +6,19 @@ build
 out
 CMakeSettings.json
 compile_commands.json
-.cache/
\ No newline at end of file
+.cache/
+{
+  "configurations": [
+    {
+      "name": "x64-Debug",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
+      "installRoot": "${projectDir}\\out\\install\\${name}",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": ""
+    }
+  ]
+}
```

Comment:
I mean, add the filename into this file. which could prevent add it again.

Reply:
Has been modified.